data/DiaSource.yaml

```diff
@@ -131,23 +175,53 @@ funcs:
     # These values below work but need a new functor for converting pixel^2
     # units to arcsec^2 DM-
     Ixx:
-        functor: Column
-        args: slot_Shape_xx
+        functor: ConvertPixelSqToArcsecondsSq
+        args:
+            - slot_Shape_xx
+            - base_LocalWcs_CDMatrix_1_1
+            - base_LocalWcs_CDMatrix_1_2
+            - base_LocalWcs_CDMatrix_2_1
+            - base_LocalWcs_CDMatrix_2_2
     Iyy:
-        functor: Column
-        args: slot_Shape_yy
+        functor: ConvertPixelSqToArcsecondsSq
+        args:
+            - slot_Shape_yy
+            - base_LocalWcs_CDMatrix_1_1
+            - base_LocalWcs_CDMatrix_1_2
+            - base_LocalWcs_CDMatrix_2_1
+            - base_LocalWcs_CDMatrix_2_2
     Ixy:
-        functor: Column
-        args: slot_Shape_xy
+        functor: ConvertPixelSqToArcsecondsSq
+        args:
+            - slot_Shape_xy
+            - base_LocalWcs_CDMatrix_1_1
+            - base_LocalWcs_CDMatrix_1_2
+            - base_LocalWcs_CDMatrix_2_1
+            - base_LocalWcs_CDMatrix_2_2
     # Icov not implemented
     IxxPsf:
-        functor: Column
-        args: slot_PsfShape_xx
+        functor: ConvertPixelSqToArcsecondsSq
+        args:
+            - slot_Shape_xx 
```

Comment:
This looks like it would be identical in value to `Ixx`: Shouldn't the `IxxPsf` use an explicitly PSF-like shape? `slot_Shape_xx` is not necessarily PSF-like. I see you used `slot_PsfShape_xy` for `IxyPsf` below.

Reply:
Thanks for the catch. Was that way due to a bad copy paste job. Now fixed.